src/TraceEvent/Parsers/GCDynamicTraceEventParser.cs

```diff
@@ -0,0 +1,586 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+using Microsoft.Diagnostics.Tracing.Parsers.GCDynamic;
+
+namespace Microsoft.Diagnostics.Tracing.Parsers
```

Comment:
it would be great if you could please put a blurb that just explains how a dynamic event is interpreted differently from a traditional event. thanks!

Reply:
That's a good idea.  I'd also cover the fact that there are two paths - one that is used when parsing events directly from a raw trace (e.g. etl or nettrace) and a slightly different mechanism that gets used when dealing with a TraceLog.  Also worth calling out the scheme being used for event IDs - that we're using unused event IDs to allow these events to be treated just like any other when it comes to pulling them from a TraceLog, showing up in the Events view, etc.

Reply:
@brianrob: Added - please let me know if the comments are apt and descriptive enough.

Reply:
This looks good, thanks.